cmd/openshift-install/testdata/agent/image/manifests/default_manifests.txt

```diff
@@ -101,7 +101,7 @@ metadata:
   name: openshift-was not built correctly
   namespace: cluster0
 spec:
-  releaseImage: registry.ci.openshift.org/origin/release:4.14
+  releaseImage: registry.ci.openshift.org/origin/release:4.15
```

Comment:
Can we just do this? https://github.com/openshift/installer/blob/master/cmd/openshift-install/testdata/agent/image/manifests/default_manifests.txt#L108